src/clr-angular/forms/combobox/_combobox.clarity.scss

```diff
@@ -1,69 +1,188 @@
+        background-color,
+        clr-ng-select-pill-background-color,
+        $clr-ng-select-pill-background-color,
+        $clr-use-custom-properties
+      );
+      border-width: $clr_baselineRem_1px;
+      border-style: solid;
+      @include css-var(
+        border-color,
+        clr-ng-select-pill-border-color,
+        $clr-ng-select-pill-border-color,
+        $clr-use-custom-properties
+      );
+
+      padding: 0 $clr_baselineRem_4px;
+
+      .clr-combobox-pill-content {
+        @include css-var(
+          color,
+          clr-ng-select-pill-font-color,
+          $clr-ng-select-pill-font-color,
+          $clr-use-custom-properties
+        );
+        font-size: $clr-typography-xxsmalltext;
+        font-weight: $clr-font-weight-regular;
+        padding: 0 $clr_baselineRem_8px 0 $clr_baselineRem_4px;
+      }
+    }
+
+    .clr-combobox-input-wrapper {
+      border: none;
+      background: none;
+    }
   }
 
   .clr-combobox-trigger {
     //Dimensions
-    @include equilateral($clr-ng-select-trigger-dimensions);
-    padding: 0;
-    margin: 0;
+    width: $clr-ng-select-trigger-dimensions;
+    margin: auto;
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    right: 0;
 
     //Other
     background: none;
     border: none;
     @include css-var(color, clr-ng-select-trigger-color, $clr-ng-select-trigger-color, $clr-use-custom-properties);
     cursor: pointer;
     outline: none;
-    transform: translateX(-1 * $clr-ng-select-trigger-dimensions);
   }
 
   .clr-options {
     @include generate-dropdown-menu();
+    position: static;
+
+    max-height: $clr_baselineRem_10;
+    overflow-y: scroll;
 
     visibility: visible;
+
+    .clr-options-loading {
+      padding: $clr_baselineRem_0_125 $clr_baselineRem_1;
+
+      .clr-options-loading-text {
+        padding-left: $clr_baselineRem_0_25;
+      }
+    }
+
+    .clr-options-empty-text {
+      padding-left: $clr_baselineRem_0_5;
+    }
   }
 
   .clr-option {
     @include generate-dropdown-item();
+    // Manual pseudo focus handling
+    &.clr-focus {
+      @include css-var(
+        background-color,
+        clr-dropdown-bg-hover-color,
+        $clr-dropdown-bg-hover-color,
+        $clr-use-custom-properties
+      );
+    }
   }
 
-  //NOTE: Adjust the height of the option only when its in the options menu
+  //NOTE: Adjust the height of the option only when it's in the options menu
   .clr-options .clr-option {
     @include generate-dropdown-item-height();
   }
 
-  .clr-combobox-input .clr-option {
+  .clr-options .clr-option {
     &:hover,
     &.active {
-      background: none;
+      @include css-var(color, clr-dropdown-item-color, $clr-dropdown-item-color, $clr-use-custom-properties);
     }
   }
+
+  .clr-combobox-disabled {
+    @include disabled-form-fields();
+
+    .clr-combobox-input,
+    .clr-combobox-remove-btn,
+    .clr-combobox-trigger {
+      cursor: not-allowed;
+    }
+  }
+
+  [clrFilterHighlight] b {
```

Comment:
We should have the filter highlight render a class instead so we can match on it instead of matching on the attribute.

Reply:
Done.